### ISSUE TITLE
models/algebras.js

### ISSUE THREAD

**Issue by justinbmeyer** [donejs/bitcentive#198]
- [ ] Rename to `feathers-algebra`.  
- [ ] Understand what's happening with `token`.  Can it be removed.

**Comment by marshallswain** [comment]
Yeah, `token` should be removed.  It's only ever sent during authentication.

**Comment by marshallswain** [comment]
We can make the `can-connect-feathers/service/` behavior mix in a Feathers algebra, automatically.  It would need to support the attributes on these two pages:

- https://docs.feathersjs.com/databases/pagination.html
- https://docs.feathersjs.com/databases/querying.html
